src/main/java/org/galatea/starter/service/IexClient.java

```diff
@@ -29,7 +33,19 @@
    * @param symbols stock symbols to get last traded price for.
    * @return a list of the last traded price for each of the symbols passed in.
    */
-  @GetMapping("/tops/last")
+  @GetMapping("/tops/last?token=${spring.application.iex_token}")
   List<IexLastTradedPrice> getLastTradedPriceForSymbols(@RequestParam("symbols") String[] symbols);
 
+  /**
+   * Get the historical price for each symbol passed in for the date passed in.
+   *
+   * @param symbol symbol to get historical price for.
+   * @param date (formatted YYYYMMDD) the date for which you want the price on.
+   * @return a IexHistoricalPrices objects for the given symbols.
+   */
+  @GetMapping("/stock/{symbol}/chart/{date}?token=${spring.application.iex_token}")
+  List<IexHistoricalPrices> getHistoricalPricesForSymbol(@PathVariable("symbol") String symbol,
+      @PathVariable("date") String date);
```

Comment:
Isn't there also a `range` param?

Reply:
I changed the variable name from "date" to "dateOrRange" and updated the explanation in the comments. However, no actual implementation should need to change I believe.

Reply:
Also wrote a test to confirm this functionality.